requirements_dev.txt

```diff
@@ -20,3 +20,4 @@ google-cloud-storage
 fsspec==0.8.0
 kubernetes==10.0.1
 black==19.10b0
+dacite==1.6.0
```

Comment:
We really should change this repo over to the pip-compile method of pinning packages, if you're interested in doing that at any point in the future (not for this PR).